### DISCUSSION TITLE
Recommended python version

### DISCUSSION THREAD

**Discussion by Nateu** [tomer8007/kik-bot-api-unofficial#216] [Q&A]
Hello, I'm curious to know what is the recommend python version to used with this module?
I've got some code what I want to have make use of this module but it's written in 3.10. Should I downgrade or should it all work just fine?

I'm running in a bit of a snag I don't think it's related to my python version tho as I have the same issue wen trying the examples on 3.9. However I'll start a separate thread for that as a comment to the code that has recently changed as I think that's where the problem is.

**Comment by tomer8007** [comment]
I'm running `echo_bot.py` successfully with python 3.7 on Windows, but it looks like you found a bug where it doesn't work on some configurations, so please open an issue for this.

P.S What is your installed `protobuf` package version?
